aspnetcore/fundamentals/minimal-apis/route-handlers.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Route handlers in Minimal API applications
+author: rick-anderson
+description: Learn how to handle requests in Minimal API applications.
+ms.author: riande
+monikerRange: '>= aspnetcore-7.0'
+ms.date: 10/31/2022
+uid: fundamentals/minimal-apis/route-handlers
+---
+
+# Route Handlers
+
+A configured `WebApplication` supports `Map{Verb}` and <xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions.MapMethods%2A>:
```

Comment:
```suggestion
A configured `WebApplication` supports `Map{Verb}` and <xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions.MapMethods%2A>:
```

Clarify what "Verb" means here?

Reply:
This isn't new, but I agree it can be improved. It's unfortunate this is one of the few places where I was not able to deduplicate. It would be really easy to fix here but not on the cheat sheet if you hadn't just worked this. Still we've deduped most of the content. What do we think about this?

```suggestion
A configured `WebApplication` supports `Map{Verb}` and <xref:Microsoft.AspNetCore.Builder.EndpointRouteBuilderExtensions.MapMethods%2A> where `{Verb}` is a camel-cased HTTP method like `Get`, `Post`, `Put` or `Delete`:
```